code/modules/uplink/uplink_items/uplink_roles.dm

```diff
@@ -172,6 +172,14 @@
 	restricted_roles = list("Cook", "Janitor")
 	item = /obj/item/ammo_box/magazine/m45/kitchengun
 
+/datum/uplink_item/role_restricted/bullet_sponge
+	name = "Bullet Sponge"
+	desc = "A seemingly magic sponge that is able to remove spent bullet case from any crime scene!"
+	cost = 1
+	surplus = 20
+	restricted_roles = list("Cook", "Janitor", "Assistant")
+	item = /obj/item/soap/sponge/bullet
+
```

Comment:
You said this was admin only. Also uplink bloat. Remove.

Reply:
What...